tests/admin_filters/models.py

```diff
@@ -46,8 +46,20 @@ def __str__(self):
         return self.description
 
 
+class Office(models.Model):
+    code = models.CharField(max_length=3, unique=True)
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        ordering = ('-pk',)
```

Comment:
Can we order by `code`? It would be more predictable and readable in tests.

Reply:
I changed models to cover all test cases without broking existing ones.